.github/workflows/main_win.yaml

```diff
@@ -172,11 +178,13 @@ jobs:
         config: [devel]
         include: ${{ fromJson(needs.initialize.outputs.includes) }}
     steps:
-      - name: Checkout
-        uses: kaidokert/checkout@v3.5.999
+      - name: Clean up
+        run: rm -rf ${{ github.workspace }}/win-win32_devel
```

Comment:
This is good, but i do recommend adding `-aoa` flag to the 7-zip unzipping as well just to be sure we don't trip on that again.